### DISCUSSION TITLE
How to remove the bookmark search?

### DISCUSSION THREAD

**Discussion by Henu3duka** [Aris-t2/CustomCSSforFx#612] [Q&A]
How to remove the bookmark search on top of the bookmark droplist?
FF116.0.3

**Comment by glennpc** [comment]
Please see https://github.com/Aris-t2/CustomCSSforFx/discussions/606#discussioncomment-6356986

**Comment by Henu3duka** [comment]
Yes! in 4.4.1 works fine! Thanx
